src/renderer/components/home.vue

```diff
@@ -1,37 +1,43 @@
 <template>
-    <div class="grid-container fluid text-center">
-        <div class="grid-x grid-padding-y">
-          <div id="challenge" class="cell small-2"></div>
-          <div class="cell small-8">
-              <h2>Tournament Management System</h2>
-              <h4>v2018.2.0</h4>
-          </div>
-          <div id="fll" class="cell small-2"></div>
+    <div class="ui center aligned grid container">
+        <div class="row">
+            <div id="challenge" class="three wide column"></div>
```

Comment:
As you can see, there is an inherited problem with displaying the logo of the challenge/FLL directly on the background, 1. becuase how it looks and 2. because of _FIRST_ official rules for displaying their logos. Please use `segment`s here as well.